app/editor/src/features/admin/reports/ReportForm.tsx

```diff
@@ -70,49 +83,66 @@ export const ReportForm: React.FC = () => {
   const [sendTo, setSendTo] = React.useState('');
   const [active, setActive] = React.useState('report');
   const [preview, setPreview] = React.useState<IReportPreviewModel>();
+  const [templateOptions, setTemplateOptions] = React.useState<IOptionItem[]>(
+    getSortableOptions(reportTemplates, [new OptionItem('[New Template]', 0)]),
+  );
 
   const reportId = Number(id);
   const reportTypeOptions = getEnumStringOptions(ReportTypeName);
 
   React.useEffect(() => {
-    findUsers({}).catch(() => {
-      // Handled already.
-    });
+    if (!users.items.length)
+      findUsers({}).catch(() => {
+        // Handled already.
+      });
+    findAllReportTemplates()
```

Comment:
Report templates are now separated from a report.  We can share a single template with many reports.